lib/cisco_node_utils/cmd_ref/bfd_global.yaml

```diff
@@ -0,0 +1,27 @@
+# bfd_global
+---
+_exclude: [ios_xr]
+
+_template:
+  get_command: "show running-config bfd all"
+  context: ~
+
+echo_interface:
+  kind: int
+  get_value: '/^bfd echo-interface loopback(\d+)$/'
+  set_value: '<state> bfd echo-interface loopback <intf>'
```

Comment:
I see where you're going with this but it seems wrong to specify the interface as just a number; i.e.

```
  echo_interface => '10'                 # Bad
  echo_interface => 'loopback10'  # Good
```